pkg/controllers/deprovisioning/controller.go

```diff
@@ -124,10 +125,17 @@ func (c *Controller) Reconcile(ctx context.Context, _ reconcile.Request) (reconc
 	}
 	// Attempt different deprovisioning methods. We'll only let one method perform an action
 	isConsolidated := c.cluster.Consolidated()
-	for _, d := range c.deprovisioners {
+	for i, d := range c.deprovisioners {
 		c.recordRun(fmt.Sprintf("%T", d))
 		success, err := c.deprovision(ctx, d)
 		if err != nil {
+			var validationErr *ValidationError
+			// if the consolidation action fails validation, continue to the next deprovisioner
+			// when it's not the last one in the chain
+			if i != len(c.deprovisioners)-1 && errors.As(err, &validationErr) {
```

Comment:
Thoughts on logging inside the deprovisioner, return success=false, error=nil, and avoid the continue hack in this context?